content/kotlin/concepts/strings/terms/dropLast/dropLast.md

```diff
@@ -0,0 +1,45 @@
+---
+Title: 'dropLast()'
+Description: 'Returns a string with n characters removed.'
+Subject:
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'Data Types'
+  - 'Characters'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+The Kotlin method, **`.dropLast()`** removes the last character of a string or the last `n` number of charcters from a string. This method does not modify the original list, it creates and returns a new list.
+
+## Syntax
+
+```pseudo
+string.dropLast()
```

Comment:
The syntax is not correct, there needs to be a parameter that is passed into the function.